### DISCUSSION TITLE
Welcome to rope Discussions!

### DISCUSSION THREAD

**Discussion by lieryan** [python-rope/rope#362] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask for help using editors that uses rope.
  * Ask for help integrating rope into existing editors/tools.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phill-tornroth** [comment]
Hi @lieryan ! I've always been interested in rope at a distance but haven't really leaned into learning more about how it works until the past couple of weeks. I'd really like to continue to learn about the library and contribute to it's documentation and functionality. I maintain a number of large/older python codebases so have plenty of motivation for tools like this one to be great. Is github Discussions the right place to do that going forward or is there another more active forum for contributors?

**Reply by lieryan** [reply]
Hi @phill-tornroth, thank you for your interest and welcome to Rope 😃.

Yes, Github Discussions is right now the best place for this.

If there is anything that could be called "more active forum" for contributors, that would be the Github Issue Tracker :sweat_smile:. But not all discussions are suitable to be in issue tracker format, so the way I envision it, Github Discussions is the space for those other things that don't fit an issue tracker.

As you can see, this is not a very active forum yet, but I'm currently trying to build this space out for anyone that are interested in Rope and open source Python refactoring tooling in any way, so it's still a work in progress.

There used to be a Google Groups mailing list and Freenode IRC for Rope, but those got decommissioned some time ago (not that they had been very active spaces to begin with), so Github Discussions is currently the best place for Rope's community to engage with each other.

That said, I'm not an experienced community builder, so I will take any help I can to build up this space; and if there is someone who are interested in taking a role to do more sophisticated community building and they believe there is another platform that would be more suitable for it, then I'd be happy to help move this community space elsewhere, but for the foreseeable future Discussions is the best space.
